src/zmq/CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ target_compile_definitions(bitcoin_zmq
 target_link_libraries(bitcoin_zmq
   PRIVATE
     core
-    leveldb
```

Comment:
Totally unrelated to CMake/this PR, but univalue is an unfortunate dependency for zmq. Seems to me it'd make sense to move `zmq/rpc.cpp` to `rpc/zmq.cpp` :)